openmdao/docs/pre_tag.py

```diff
@@ -0,0 +1,94 @@
+#pre_tag.py
+#A script that finds occurrences of the .. tags:: directive
+#and sets up the structure of the tags directory.  One file
+#is created for each subject tag, that file contains links to
+#each instance of the tag throughout the docs.
+
+import sys, os, shutil, re
+
+def make_tagdir():
+    #clean up tagdir, create tagdir, return tagdir
+    dir = os.path.dirname(__file__)
+    tagdir = os.path.join(dir, "tags")
+
+    if os.path.isdir(tagdir):
+      shutil.rmtree(tagdir)
+
+    if not os.path.isdir(tagdir):
```

Comment:
this isdir check doesn't seem necessary since you already removed the dir if it was there